script/playercomponent.js

```diff
@@ -29,7 +29,8 @@ define(
         windowType,
         mediaKind,
         playbackElement,
-        bigscreenPlayerData.media.isUHD
+        bigscreenPlayerData.media.isUHD,
+        bigscreenPlayerData.media.mseCustomisation
```

Comment:
Is there a chance we'd want to set configuration options for non-MSE strategies?

Reply:
I would honestly say that is never going to happen but down to change the name to something more generic. playerSettings?

Reply:
Yeah I like `playerSettings`.

It's just on the off chance we want to influence some configuration across multiple or a different strategy. Like you said, likely to never happen but might be worth making generic now in case.